### DISCUSSION TITLE
Кнопка "Предложить координаты" / "Suggest coordinates" button

### DISCUSSION THREAD

**Discussion by kabalin** [PastVu/pastvu#640] [Ideas / Идеи и предложения]
Кнопка "Предложить координаты" для неопознанных снимков, которая бы позволила выставить точку на карте и отправить. Модератор мог бы видеть очередь предложенных записей, принимать или отклонять. Если это связать с системой рейтинга (#636), за число принятых координат начислялся бы рейтинг определенного типа (формулировка "следопыт" предложена в комментариях https://pastvu.com/news/141?hl=comment-2978112), либо можно бы было эту информацию отображать в статистике пользователя.

**Comment by paul-k-pastvu** [comment]
Да, но я бы расширил. Есть много разных ситуаций, в которых нужно "позвать модератора", и предложение точки/направления - только одна из них. А ещё:
- изменение названия
- изменение датировок
- уточнение авторства
- пометка фото как варианта или дубля другого фото
- пометка комментариев как нарушающих правила и требующих удаления

**Comment by paul-k-pastvu** [comment]
Т.е. кнопки могут быть и разные, но система д.б. общая
